web/src/views/Dashboards/Dashboards.vue

```diff
@@ -276,19 +276,17 @@ export default defineComponent({
       });
     });
 
-    const removeDashboard = async () => {
+    const deleteDashboard = async () => {
       if (selectedDelete.value) {
         const dashboardId = selectedDelete.value.id;
         await dashboardService
           .delete(store.state.selectedOrganization.identifier, dashboardId)
           .then((res) => {
-            const dashboardList = JSON.parse(
-              JSON.stringify(toRaw(store.state.allDashboardList))
```

Comment:
We need an object. We have it. Case closed.

`JSON.parse(JSON.stringify(obj))` pattern feels generally wrong to me. I'm not a JavaScript expert though, so maybe it is useful, [if controversial](https://medium.com/@pmzubar/why-json-parse-json-stringify-is-a-bad-practice-to-clone-an-object-in-javascript-b28ac5e36521), in some cases.